### ISSUE TITLE
Advertisements

### ISSUE THREAD

**Issue by Lectric1** [jonzenor/ChristianGuilds#128]
Go to bed but also advertisements

**Comment by Lectric1** [comment]
Marketing is gonna be essential in getting this off the ground,

**Comment by jonzenor** [comment]
Agreed. I don't have a big budget for this project so I can't pay for most of the advertising... I have a few connections in the Christian gaming community so I am banking on that help, but I don't know how far that will get me in the long run.

I've tried paid advertisements in the past but have gotten very little, or almost no ROI on those...

**Comment by Lectric1** [comment]
Perhaps you need better adds

**Comment by jonzenor** [comment]
Maybe... but the time it takes to research that and do AB testing for adds and to tweak things to optimum is time that I could be programming. 😃
